user.tf

```diff
@@ -1,14 +1,25 @@
 resource "random_password" "sql_user" {
-  for_each = local.users
+  for_each = merge(
+    local.mysql_users,
+    local.postgres_users,
+  )
```

Comment:
How can we distinguish mysql and postgresql users with the same name?

Reply:
Nevermind, those would be in different "instances".

Reply:
Doing it this way makes it easier to refer to the passwords elsewhere, most notably in the `outputs {` section and everywhere else those outputs are used.